pkg/email/new_document.go

```diff
@@ -0,0 +1,103 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"path"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+type newDocument struct {
+	DocumentType string
+	RequestName  string
+	RequestLink  string
+}
+
+func stringifyDocumentType(commonType models.AccessibilityRequestDocumentCommonType, otherType string) string {
+	switch commonType {
+	case models.AccessibilityRequestDocumentCommonTypeAwardedVpat:
+		return "Awarded VPAT"
+	case models.AccessibilityRequestDocumentCommonTypeTestPlan:
+		return "Test plan"
+	case models.AccessibilityRequestDocumentCommonTypeRemediationPlan:
+		return "Remediation plan"
+	case models.AccessibilityRequestDocumentCommonTypeTestingVpat:
+		return "Testing VPAT"
+	case models.AccessibilityRequestDocumentCommonTypeTestResults:
+		return "Test results"
+	case models.AccessibilityRequestDocumentCommonTypeOther:
+		return otherType
+	default:
+		return ""
```

Comment:
Would it be better to return some kind of generic here instead of an empty string? We're probably never going to get to this given all our other type checks in the system, so I think this works, too.